### PR TITLE
Textcomp package

### PR DESCRIPTION
**Pull request by zorkow**
The textcomp package. Works with both `TexParser` and `TextParser`.
Note that in the `TexParser` elements are not combined into a single `text` element. Likewise, `oldstyle` elements are never combined. This could be done with a `postFilter` but I don't think it is worth the effort for now.

### CODE REVIEW

File: ts/input/tex/textcomp/TextcompMappings.ts [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  'textnaira':           ['Insert', '\u20A6'],
+  'textpeso':            ['Insert', '\u20B1'],
+  'textsterling':        ['Insert', '\u00A3'],
+  'textwon':             ['Insert', '\u20A9'],
+  'textyen':             ['Insert', '\u00A5'],
+
+  // Table 15: textcomp Legal Symbols
+  'textcircledP':        ['Insert', '\u2117'],
+  'textcompwordmark':    ['Insert', '\u200C'],
+  'textcopyleft':        ['Insert', '\u1F12F'],
```
Comment (dpvc) [inline ts/input/tex/textcomp/TextcompMappings.ts:99]: This needs to be `\u{1F12F}` otherwise it is U+1F12 followed by an F.

**Review by dpvc — APPROVED** [review]
One small correction, otherwise looks good.

File: ts/input/tex/textcomp/TextcompConfiguration.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import {Configuration} from '../Configuration.js';
+import './TextcompMappings.js';
+
+
+export const TextcompConfiguration = Configuration.create(
+  'textcomp', {
+    parser: 'tex',
```
[the remark this answers is not in the document]
Comment (zorkow) [reply ts/input/tex/textcomp/TextcompConfiguration.ts]: Removed. That was leftover when it was supposed to be `parser: 'text'`.

File: ts/input/tex/textcomp/TextcompMappings.ts (hunk 1 above)
Comment (zorkow) [reply ts/input/tex/textcomp/TextcompMappings.ts]: Thanks for catching this. Of course it still produces an error in chtml until PR #723 is merged.

**Pull request merged**